src/Chapter06/Listing06.39.SpecifyingRequiredMembersWithinTheObjectInitializer.cs

```diff
@@ -1,27 +1,23 @@
-#if NET7_0_OR_GREATER
-
-#if COMPILEERROR // EXCLUDE
-using AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_38;
-#endif // COMPILEERROR // EXCLUDE
+#if NET7_0_OR_GREATER // EXCLUDE
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_39;
 
 using System.Runtime.CompilerServices;
-
-namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_39;
+using Listing06_38;
 
 public class Program
 {
     public static void Main()
     {
-        #if COMPILEERROR // EXCLUDE
         #region INCLUDE
         // Error CS9035:
         // Required member 'Book.Isbn' must be set in the object
         // initializer or attribute constructor
+        #if COMPILEERROR // EXCLUDE
         Book book = new() { Title= "Essential C#" };
+        #endif // COMPILEERROR // EXCLUDE
 
         // ...
         #endregion INCLUDE
-        #endif // COMPILEERROR // EXCLUDE
-      }
+    }
 }
-#endif // NET7_0_OR_GREATER
+#endif // NET7_0_OR_GREATER // EXCLUDE
```

Comment:
This looks a bit awkward to have dual `//` in the same line.

Reply:
I think the parser right now looks for `// EXCLUDE` explicitly if I am reading the regex right https://github.com/IntelliTect-dev/EssentialCSharp.Tooling/blob/12dffde4ff4dfbc27f0c4cd6b49b8bad8bb15cbb/Parser/CSharp/CSharpParser.cs#L431-L432C14

Reply:
Yuck, yea you are correct, but this is now one more case where our "source" data is going to start to look awkward for the sake of the parser. I am fine leaving this as is, but still making me think about better ways to share the code.